decorate-angular-cli.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * This file decorates the Angular CLI with the Nx CLI to enable features such as computation caching
```

Comment:
I have no idea, what this file is? Why isn't it part of nx?

Reply:
This file is created by nx. This is the patch file for ng.js and after that the ng command works like nx